samples/ar-view-node/src/main/res/layout/view_node_circle_outlined.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
```

Comment:
Can the `ImageView` be a root `View`?

Reply:
I tested it and the result is strange. The circle drawable is not centered anymore. 

LinearLayout as root
![view_node_circle_outlined_ll_as_root](https://github.com/SceneView/sceneview-android/assets/3974162/61ed4b03-51c4-4aaa-879b-f8ef13a7daa6)

ImageView as root
![view_node_circle_outlined_iv_as_root](https://github.com/SceneView/sceneview-android/assets/3974162/b930ddb6-af1c-4e08-8334-94c048ac96d1)